WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -111,7 +111,7 @@ import WordPressAuthenticator
         }
 
         if debugKey == ApiCredentials.debuggingKey(), debugType == "crashlytics_crash" {
```

Comment:
Should we change this to `sentry_crash`?

Reply:
We could – I didn't, because I wanted to keep everything backwards-compatible with folks' existing setup as much as possible.

If we do change it, I'd like to do something non-product-specific so that we don't have to change it again? Something like 'force_crash`?

Reply:
👍 I'm fine with either. Keeping it the same for compatibility makes sense so I'll leave it up to you.

Reply:
Adjust it for consistency. The only remaining reference to Crashlytics is https://github.com/wordpress-mobile/WordPress-iOS/blob/4d20605802142fda99c44765ac776cd9a53c6152/WordPress/Classes/Utility/Sentry/WPCrashLogging.swift#L4, but I don't want to lose the user's preference in this transition (or write transition logic that could cause that) so IMHO it makes sense for that reference to remain.